api/AbortPaymentEvent.json

```diff
@@ -5,61 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbortPaymentEvent",
         "support": {
           "chrome": {
-            "version_added": "61",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "#service-worker-payment-apps",
-                "value_to_set": "Enabled"
-              }
-            ]
+            "version_added": "70"
           },
           "chrome_android": {
-            "version_added": "61",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "#service-worker-payment-apps",
-                "value_to_set": "Enabled"
-              }
-            ]
+            "version_added": "70"
           },
           "edge": {
-            "version_added": "≤79",
-            "flags": [
-              {
-                "type": "preference",
-                "name": "#service-worker-payment-apps",
-                "value_to_set": "Enabled"
-              }
-            ]
+            "version_added": "79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": false
           },
           "firefox_android": {
-            "version_added": null
+            "version_added": false
           },
           "ie": {
             "version_added": false
           },
           "opera": {
-            "version_added": false
+            "version_added": "57"
           },
           "opera_android": {
-            "version_added": false
+            "version_added": "49"
           },
           "safari": {
-            "version_added": null
+            "version_added": false
           },
           "safari_ios": {
-            "version_added": null
+            "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": false
+            "version_added": true
           },
           "webview_android": {
-            "version_added": false
+            "version_added": "70"
```

Comment:
Are you sure about this and the other webview_android changes? It seems a bit implausible that the event interfaces should be supported when the payment APIs that would cause the events to be fired are not.

Reply:
As surprising as it is, some interfaces were exposed in WebView for the Payment APIs.  I have no doubt that this was a mistake and they weren't supposed to be exposed, but I wanted to go for accuracy.  I can keep it all `false` if preferred.

Reply:
That sort of mistake happens from time to time, unfortunately. Since it will look like a mistake to anyone looking at the data or MDN pages, can you leave it as false in this PR? We can make the changes, but then add notes as well. Doing that in a separate PR would be best I think.